### PR TITLE
Register Subdomain: elarifamine

### PR DESCRIPTION
**Pull request by AmineArif1**
Added the domain: elarifamine

### CODE REVIEW

File: domains/elarifamine.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "AmineArif1",
+        "email": "elarifamine1@gmail.com"
+      },
+
+      "record": {
+        "CNAME": "aminearif1.github.io"
```
Comment (WilliamDavidHarrison) [inline domains/elarifamine.json:9]: 404?

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Review by wdhdev — APPROVED** [review]

File: domains/elarifamine.json (hunk 1 above)
Comment (AmineArif1) [reply domains/elarifamine.json]: Hey William , 
Sorry for that , i updated the link!
thanks.

**Review by wdhdev — DISMISSED** [review]

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/elarifamine.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "AmineArif1",
+        "email": "elarifamine1@gmail.com"
+      },
+
+      "record": {
+        "CNAME": "www.elarifamine.tk"
```
[the remark this answers is not in the document]
Comment (AmineArif1) [reply domains/elarifamine.json:9]: Hey @MaskDuck ,
It is working now, sorry about that.

**Review by wdhdev — APPROVED** [review]
Site reachable + has valid content

**Pull request closed without merging**